src/packages/excalidraw/CHANGELOG.MD

```diff
@@ -18,6 +18,7 @@ The change should be grouped under one of the below section and must contain PR
 - Add Cut to menus [#2511](https://github.com/excalidraw/excalidraw/pull/2511)
 - More Arrowheads: dot, bar [#2486](https://github.com/excalidraw/excalidraw/pull/2486)
 - Support CSV graphs and improve the look and feel [#2495](https://github.com/excalidraw/excalidraw/pull/2495)
+- Add zoom to selection [#2522](https://github.com/excalidraw/excalidraw/pull/2522)
```

Comment:
@mwufi can you move this to the top under features? so the latest features are on top.

Reply:
Oh, okay :)

Reply:
Added a little note as well, to make this clearer